.github/workflows/e2e.yml

```diff
@@ -15,75 +15,86 @@
 name: Build e2e tests
 
 on:
-  push:
-    branches: [ github-actions-salesforce-automation ]
-  pull_request:
-    branches: [ github-actions-salesforce-automation ]
-    types: [opened, synchronize, reopened, labeled]
-  workflow_dispatch:
+  workflow_run:
+    workflows:
+      - Trigger build
+    types:
+      - completed
 
 jobs:
   build:
     runs-on: k8s-runner-e2e
 
-    # We allow builds:
-    # 1) When triggered manually
-    # 2) When it's a merge into a branch
-    # 3) For PRs that are labeled as build and
-    #  - It's a code change
-    #  - A build label was just added
-    # A bit complex, but prevents builds when other labels are manipulated
-    if: >
-      github.event_name == 'workflow_dispatch'
-      || github.event_name == 'push'
-      || (contains(github.event.pull_request.labels.*.name, 'build')
-         && (github.event.action != 'labeled' || github.event.label.name == 'build')
-         )
+    if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
+
     steps:
-      # Pinned 1.0.0 version
```

Comment:
pls revert this change

Reply:
Added the comment back (18bd6bbba7443d9e5c842d005d05000f3f295928)